integrationExamples/gpt/optimeraRtdProvider_example.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="icon" type="image/png" href="/favicon.png">
+  <script async src="//www.googletagservices.com/tag/js/gpt.js"></script>
```

Comment:
this is the old gpt

Reply:
Hi @patmmccann, good to meet you. I updated the tag.